_data/simple-icons.json

```diff
@@ -11162,6 +11162,11 @@
             "source": "https://www.raycast.com/press",
             "guidelines": "https://www.raycast.com/press"
         },
+        {
+            "title": "Raylib",
+            "hex": "000000",
+            "source": "https://github.com/raysan5/raylib/tree/master/logo"
```

Comment:
The source if from GitHub has to be the permalink for the exact file instead of a folder. That is why linter is failing.